File: pgm/ABC_Visual_Basic/ABC_168_B.vb
' Imports System.Collections.Generic
' Imports System.Text.RegularExpressions

Class Main
    Public Shared Sub Main()
        Dim K As Integer
        DIm S As String
        K = Console.ReadLine()
        S = Console.ReadLine()
        If S.Length <= K Then
            Console.WriteLine(S)
        Else
            Console.WriteLine(S.Substring(0, K) & "...")
        End If
        Return
    End Sub
End Class

File: pgm/ABC_C#/ABC_168_B.cs
using System;
// using System.Collections.Generic;
// using System.Linq;

class Program
{
    static void Main()
    {
        int K = int.Parse(Console.ReadLine());
        string S = Console.ReadLine();
        if (S.Length <= K)
            Console.WriteLine(S);
        else
            Console.WriteLine(S.Substring(0, K) + "...");
        return;
    }
}